pkg/activator/throttler.go

```diff
@@ -271,3 +295,64 @@ func (t *Throttler) endpointsDeleted(obj interface{}) {
 	revID := RevisionID{ep.Namespace, revisionName}
 	t.Remove(revID)
 }
+
+type infiniteBreaker struct {
+	mu          sync.RWMutex
+	broadcast   chan struct{}
+	concurrency int32
```

Comment:
I think it would be helpful to document the values we expect this to take on.  It took me a while to follow the code...

AIUI, `concurrency` is set to `0` when scaled to zero and `1` when scaled to any positive number.  When `concurrency` is `0` then `broadcast` is an open channel on which we can block.  When `concurrency` goes from `0` to `1` the channel is closed, immediately unblocking all active `Maybe` calls for this revision?

I am betting that at some point we are going to want to build in some sort of backpressure here or ramp up, but this is better than the current cap.  If operators want another behavior then we possibly want to add a way to specify default container concurrency.